examples/python/live_depth_sensor/main.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+"""
+A minimal example of streaming frames live from an intel realsense depth sensor.
+
+NOTE: this example currently runs forever and will eventually exhaust your
+system memory. It is advised you run an independent rerun viewer with a memory
+limit:
+```
+rerun --memory-limit 4GB
+```
+
+And then connect using:
+```
+python examples/python/live_depth_sensor/main.py --connect
+```
+
+"""
+from __future__ import annotations
+
+import argparse
+
+import numpy as np
+import pyrealsense2 as rs
+import rerun as rr  # pip install rerun-sdk
+
+
+def run_realsense(num_frames: int | None) -> None:
+    # Visualize the data as RDF
+    rr.log_view_coordinates("realsense", xyz="RDF", timeless=True)
+
+    # Open the pipe
+    pipe = rs.pipeline()
+    profile = pipe.start()
+
+    # Get and log depth exstrinsics
+    depth_profile = profile.get_stream(rs.stream.depth)
+    depth_intr = depth_profile.as_video_stream_profile().get_intrinsics()
+
+    rr.log_pinhole(
+        "realsense/camera/depth",
+        child_from_parent=np.array(
+            (
+                (depth_intr.fx, 0, depth_intr.ppx),
+                (0, depth_intr.fy, depth_intr.ppy),
+                (0, 0, 1),
+            ),
+        ),
+        width=depth_intr.width,
+        height=depth_intr.height,
+        timeless=True,
+    )
+
+    # Get and log color extrinsics
+    rgb_profile = profile.get_stream(rs.stream.color)
+
+    depth_extr = depth_profile.get_extrinsics_to(rgb_profile)
+    rr.log_transform3d(
+        "realsense/camera/ext",
+        transform=rr.TranslationAndMat3(
+            translation=depth_extr.translation, matrix=np.reshape(depth_extr.rotation, (3, 3))
+        ),
+        from_parent=True,
```

Comment:
Sort of if you strictly follow the API. But that's problematic because "depth" is on the far side of a pinhole transform so we can't use that.

I could log as something like
 - realsense/depth_origin/depth_image
 - realsense/depth_origin/rgb_origin/rgb_image

But I don't really find that nesting one camera inside the other adds much clarity.

From a driver perspective camera is really "under" the other.  Practically speaking they're siblings, but the  driver doesn't expose an API to an abstract origin. We could have just as easily called the reverse API though (`rgb_profile.get_extrinsics_to(depth_profile)`).  I think of this as saying that we (arbitrarilly) define the origin of the realsense space to be the same as the child (depth).

I don't love having "camera/ext" in there though either way so will add some comments and clean up.